internal/common/db.go

```diff
@@ -51,8 +60,14 @@ const (
 	SkipDeletedRecords    DeleteRecordsState = false
 )
 
-func LoadHostsFromDB(db *gorm.DB, conditions ...interface{}) *gorm.DB {
-	return db.Preload("Hosts", conditions...)
+const (
+	HostsTable = "Hosts"
```

Comment:
This isn't the table name (it's `hosts`). Maybe this should be called `HostsModelName`?
Even better is there somewhere we can get this from in the gorm model?

Reply:
I was following gorm semantics https://gorm.io/docs/preload.html
But maybe it's case insensitive.

Regarding using `models` I didnt find anything. Let me know if you do.

Reply:
The doc you linked doesn't call the capitalized name a "table name" anywhere.
Also those are the docs for the new version, so be careful. https://pkg.go.dev/github.com/jinzhu/gorm#DB.Preload is the documentation for the version we use.

Also, after a quick search there's a [whole file on naming strategies](https://github.com/jinzhu/gorm/blob/v1.9.12/naming.go) that looks like it could apply.

But I suppose this is fine for now. We might want to make this more dynamic in the future if we find ourselves changing the list of "sub tables" frequently. https://github.com/jinzhu/gorm/blob/v1.9.16/model_struct.go is worth a look for this kind of thing I think